functions.php

```diff
@@ -349,6 +349,24 @@ function ($urls, $post_id) {
     2
 );
 
+// Add filters to the News & Stories listing page.
+// Right now only "category" is available.
+add_action(
```

Comment:
As this function is specific to the News & Stories listing page, should we move it to the `ListingPage` class?

Reply:
I've tried that but it wasn't applied properly then, probably a priority issue?